Hush/Assets/Scenes/Menus/GameOver.unity

```diff
@@ -0,0 +1,2127 @@
+%YAML 1.1
```

Comment:
Make the buttons bigger and change the text (though you can also do this is another PR since Siobhan needs these changes)

![image](https://user-images.githubusercontent.com/35614138/143965366-5c58a81d-265a-404f-ac38-861e13e4ce7e.png)